2022-09-26 19:49:45,653 INFO    MainThread:3598 [wandb_setup.py:_flush():76] Configure stats pid to 3598
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_setup.py:_flush():76] Loading settings from /Users/vlourenco/.config/wandb/settings
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_setup.py:_flush():76] Loading settings from /Users/vlourenco/Documents/GitHub/EEG_MIB/Organized/wandb/settings
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Organized/train.py', 'program': '/Users/vlourenco/Documents/GitHub/EEG_MIB/Organized/train.py'}
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_init.py:_log_setup():461] Logging user logs to /Users/vlourenco/Documents/GitHub/EEG_MIB/Organized/wandb/run-20220926_194945-3jajfklb/logs/debug.log
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_init.py:_log_setup():462] Logging internal logs to /Users/vlourenco/Documents/GitHub/EEG_MIB/Organized/wandb/run-20220926_194945-3jajfklb/logs/debug-internal.log
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_init.py:init():495] calling init triggers
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_init.py:init():514] re-initializing run, found existing run on stack: 10xop3kf
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_run.py:_finish():1728] finishing run visilvestre/new-test/10xop3kf
2022-09-26 19:49:45,654 INFO    MainThread:3598 [jupyter.py:save_history():435] not saving jupyter history
2022-09-26 19:49:45,654 INFO    MainThread:3598 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_init.py:_jupyter_teardown():393] cleaning up jupyter logic
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_run.py:_atexit_cleanup():1931] got exitcode: 0
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_run.py:_restore():1914] restore
2022-09-26 19:49:45,654 INFO    MainThread:3598 [wandb_run.py:_restore():1920] restore done
2022-09-26 19:49:45,660 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 742
}

2022-09-26 19:49:47,072 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 742
}

2022-09-26 19:49:47,264 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 742
}

2022-09-26 19:49:48,192 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 742
}

2022-09-26 19:49:48,429 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 19128
}

2022-09-26 19:49:48,539 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 19128
}

2022-09-26 19:49:48,643 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18480
  total_bytes: 19128
}

2022-09-26 19:49:48,752 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19128
  total_bytes: 19128
}

2022-09-26 19:49:48,857 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19128
  total_bytes: 19128
}

2022-09-26 19:49:48,963 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19128
  total_bytes: 19128
}

2022-09-26 19:49:49,072 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19128
  total_bytes: 19128
}

2022-09-26 19:49:49,392 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19128
  total_bytes: 19128
}

2022-09-26 19:49:49,729 INFO    MainThread:3598 [wandb_run.py:_on_finish():2221] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19128
  total_bytes: 19128
}
local_info {
}

2022-09-26 19:49:50,838 INFO    MainThread:3598 [wandb_run.py:_footer_history_summary_info():3305] rendering history
2022-09-26 19:49:50,839 INFO    MainThread:3598 [wandb_run.py:_footer_history_summary_info():3337] rendering summary
2022-09-26 19:49:50,841 INFO    MainThread:3598 [wandb_run.py:_footer_sync_info():3261] logging synced files
2022-09-26 19:49:50,842 INFO    MainThread:3598 [wandb_init.py:init():548] starting backend
2022-09-26 19:49:50,842 INFO    MainThread:3598 [wandb_init.py:init():552] setting up manager
2022-09-26 19:49:50,844 INFO    MainThread:3598 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-09-26 19:49:50,844 INFO    MainThread:3598 [wandb_init.py:init():558] backend started and connected
2022-09-26 19:49:50,845 INFO    MainThread:3598 [wandb_run.py:_label_probe_notebook():1105] probe notebook
2022-09-26 19:49:50,845 INFO    MainThread:3598 [wandb_run.py:_label_probe_notebook():1115] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-09-26 19:49:50,845 INFO    MainThread:3598 [wandb_init.py:init():636] updated telemetry
2022-09-26 19:49:50,865 INFO    MainThread:3598 [wandb_init.py:init():667] communicating run to backend with 30 second timeout
2022-09-26 19:49:52,105 INFO    MainThread:3598 [wandb_run.py:_on_init():1982] communicating current version
2022-09-26 19:49:52,195 INFO    MainThread:3598 [wandb_run.py:_on_init():1986] got version response upgrade_message: "wandb version 0.13.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-09-26 19:49:52,195 INFO    MainThread:3598 [wandb_init.py:init():700] starting run threads in backend
2022-09-26 19:49:56,148 INFO    MainThread:3598 [wandb_run.py:_console_start():1962] atexit reg
2022-09-26 19:49:56,148 INFO    MainThread:3598 [wandb_run.py:_redirect():1820] redirect: SettingsConsole.WRAP_RAW
2022-09-26 19:49:56,148 INFO    MainThread:3598 [wandb_run.py:_redirect():1885] Wrapping output streams.
2022-09-26 19:49:56,148 INFO    MainThread:3598 [wandb_run.py:_redirect():1907] Redirects installed.
2022-09-26 19:49:56,148 INFO    MainThread:3598 [wandb_init.py:init():732] run started, returning control to user process
